src/modules/faucet/utils/index.ts

```diff
@@ -58,6 +58,10 @@ export const sendFaucet = async ({
     const dripAmount = ethers.utils.parseUnits(amount.toString(), ASTAR_TOKEN_DECIMALS).toString();
     const result = await astarApi.sendTokenTo({ to: address, dripAmount: new BN(dripAmount) });
 
+    if (network === Network.shiden) {
```

Comment:
In the Discord app, the connection is only established once at the beginning https://github.com/AstarNetwork/astar-faucet-bot/blob/main/src/app.ts#L60

But when the user requests a faucet through Discord, it will send the token to whatever network it has been connected to. https://github.com/AstarNetwork/astar-faucet-bot/blob/main/src/app.ts#L96
In other words, the network that the Discord app is connected to depends on the last UI network request.